common/crypto_utils/avalon_crypto_utils/signature.py

```diff
@@ -248,9 +247,21 @@ def generate_client_signature(
 
         encrypted_session_key_str = byte_array_to_hex_str(
             encrypted_session_key)
-        self.__encrypt_workorder_indata(
-            input_json_params, session_key,
-            session_iv, worker.encryption_key, data_key, data_iv)
+
+        logger.debug("Encrypting Workorder inData")
+        in_data_objects = input_json_params['inData']
+        updated_in_data_objects = self.__encrypt_workorder_data(
+            in_data_objects, session_key, session_iv,
+            worker.encryption_key, data_key, data_iv)
+        input_json_params['inData'] = updated_in_data_objects
```

Comment:
I think `in_data_objects` will be modified in-place by `__encrypt_workorder_data()`. So you don't need to return `updated_in_data_objects ` and reassign it

Reply:
Removed not needed re-assignment.